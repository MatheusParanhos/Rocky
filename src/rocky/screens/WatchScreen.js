import React, { Component } from "react";
import { Text, View, StyleSheet } from "react-native";
import WatchContainer from "../containers/WatchContainer";

export default class WatchScreen extends Component {
  render() {
    return (
      <View style={styles.container}>
        <WatchContainer />
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "white"
  }
});
