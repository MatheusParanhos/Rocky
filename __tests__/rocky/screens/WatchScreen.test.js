import React from 'react';
import WatchScreen from '../../../src/rocky/screens/WatchScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<WatchScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});