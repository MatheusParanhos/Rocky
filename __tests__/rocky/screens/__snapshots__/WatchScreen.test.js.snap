// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": 200,
          "justifyContent": "flex-end",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": 750,
        }
      }
    >
      <Text
        style={
          Object {
            "fontFamily": "SignatraDEMO",
            "fontSize": 50,
            "justifyContent": "center",
            "textAlign": "center",
            "width": 750,
          }
        }
      />
      <Text
        style={
          Object {
            "fontFamily": "SignatraDEMO",
            "fontSize": 50,
            "justifyContent": "center",
            "textAlign": "center",
            "width": 750,
          }
        }
      >
        
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 0.8,
          "flexDirection": "row",
          "width": 750,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "flexDirection": "row",
          }
        }
      >
        <Image
          resizeMode="contain"
          source={
            Object {
              "testUri": "../../../src/rocky/medias/Jack.png",
            }
          }
          style={
            Object {
              "height": 150,
              "width": 150,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "flex-end",
          }
        }
      >
        <Image
          resizeMode="contain"
          source={
            Object {
              "testUri": "../../../src/rocky/medias/Jade.png",
            }
          }
          style={
            Object {
              "height": 150,
              "width": 150,
            }
          }
        />
      </View>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "tomato",
          "height": 50,
          "justifyContent": "center",
          "opacity": 1,
          "width": 576.9230769230769,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontWeight": "800",
            }
          }
        >
           
          Rocky!
           
        </Text>
      </View>
    </View>
  </View>
</View>
`;
